package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dfinity/nns",
-  "version": "0.3.0",
+  "version": "0.4.0",
```

Comment:
neat! can you also bump the version in `package-lock.json`?

Reply:
Thanks, good point!